hive_agent/llm_settings.py

```diff
@@ -1,10 +1,15 @@
+import toml
 import os
 from llama_index.llms.openai import OpenAI
 from llama_index.core.settings import Settings
 
 
 def init_llm_settings():
-    model = os.getenv("MODEL", "gpt-3.5-turbo")
+    dir_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
i suggest creating a Config class that loads once and then has all the getters and setters. the class should be loaded before the agent runs so that it key configurations can be validated and errors caught much earlier